pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -497,5 +511,62 @@ func (r *Reconciler) createCheCluster(ctx context.Context, kcCfg *config.RHSSO,
 			},
 		},
 	}
-	return serverClient.Create(ctx, cheCluster)
+
+	ownerutil.EnsureOwner(cheCluster, inst)
+	if err := serverClient.Create(ctx, cheCluster); err != nil {
+		return nil, errors.Wrap(err, "failed to create che cluster resource")
+	}
+	return cheCluster, nil
+}
+
+// A
+func (r *Reconciler) reconcileExternalPostgres(ctx context.Context, inst *v1alpha1.Installation, serverClient pkgclient.Client) (*chev1.CheCluster, error) {
+	// have to do something to get postgres db values
+	// setup the postgres cr for cloud resource operator
+	postgresCredName := fmt.Sprintf("codeready-postgres-%s", inst.Name)
+
+	postgresCred := &crov1.Postgres{
```

Comment:
This can actually be simplified a bit now by using cro utils to reconcile the postgres resource -
https://github.com/integr8ly/integreatly-operator/pull/185/commits/508a9192a133ca29249cb9098a332da388c7f1b4#diff-f0feb54f2b37954540d4e8e4c7b535c5R496-R501

Reply:
:+1: